PRS1001-2000/PRS1461Afnin.xml

```diff
@@ -58,7 +58,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <persName xml:lang="gez" xml:id="n1">አፍኒን፡</persName>
                <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAfnin</persName>
                <persName xml:lang="he" xml:id="n2">אוֹפַנִּים</persName>
-               <persName xml:lang="he" corresp="n2">Ofanim</persName>
+               <persName xml:lang="he" corresp="#n2">Ofanim</persName>
```

Comment:
type="normalized"
for Hebrew too we should use Academic transcription (SBL)
and we need the English "standard" form
so I would suggest 
```
<persName xml:lang="gez" xml:id="n1">አፍኒን፡</persName>
               <persName xml:lang="gez" type="normalized" corresp="#n1">ʾAfnin</persName>

               <persName xml:lang="he" xml:id="n2">אוֹפַנִּים</persName>
               <persName xml:lang="he" type="normalized" corresp="#n2">ʾôp̲annîm</persName>
               <persName xml:lang="en" corresp="#n2">Ophanim</persName>
```

Reply:
Thank you for the explanation